test/e2e/dns.go

```diff
@@ -464,26 +464,5 @@ var _ = framework.KubeDescribe("DNS", func() {
 		pod2 := createDNSPod(f.Namespace.Name, wheezyProbeCmd, jessieProbeCmd, true)
 
 		validateTargetedProbeOutput(f, pod2, []string{wheezyFileName, jessieFileName}, "bar.example.com.")
-
-		// Test changing type from ExternalName to ClusterIP
-		By("changing the service to type=ClusterIP")
```

Comment:
I prefer just commenting this test case to removing it. And add one more comment to a) tell the reason why it is commented, b) put the issue link here c) say it will be uncommented in 1.8.

@MrHohn WDYT?

Reply:
Thanks for the comment! Yeah that sounds right to me, PTAL.